doc/source/new-project-guide/level-3-tutorial.rst

```diff
@@ -108,13 +198,12 @@ Then, you can run the code by running:
 .. code-block:: bash
 
     python proj_one/reuse_code.py
-    python proj_two/reuse_code.py
 
 
 Run tests
 ^^^^^^^^^^
 
-Notice that the ``tests`` folder contains a test file called ``test_calculator.py``. You can run the tests by running the following command:
+To run tests using ``pytest``, run the following command:
 
 .. code-block:: bash
 
```

Comment:
probably better behavior to teach is to encourage them to just run `pytest` in the top level direcotry (finds all tests suites and runs all tests), or `pytest path/to/tests` runs all tests in that directory called `tests`, or `pytest path/to/tests/test_functions.py` to just run the tests in that file.

recommendation is to only limit hte tests being run if you need to, for example, you want tests to run quickly during development or, during development, you want to avoid some tests that are failing.  Also, mention that it is bad practice to tolerate/work around failing tests.